requirements/edx/base.in

```diff
@@ -85,7 +85,7 @@ edx-rest-api-client
 edx-search
 edx-submissions
 edx-user-state-client
-edx-when
+edx-when==0.1.2
```

Comment:
Maybe add this to `constraints.txt` instead with an explanation comment?

Reply:
Looks like they released a version fixing this